cluster/src/test/java/com/hartwig/pipeline/transfer/staged/StagedOutputPublisherTest.java

```diff
@@ -107,8 +107,23 @@ public void publishesDnaTertiaryAnalysisOnVcf() throws Exception {
     }
 
     @Test
-    public void publishesGermlineTertiaryAnalysisOnGermlineVcf() throws Exception {
-        verifyGermline(".germline.vcf.gz", "reference.germline.vcf.gz");
+    public void publishesGermlineAnalysisOnGatkGermlineVcf() throws Exception {
+        verifyGermline(".germline.vcf.gz", "reference.germline.vcf.gz", "/germline_caller/");
+    }
+
+    @Test
+    public void publishesGermlineAnalysisOnGatkGermlineVcfIndex() throws Exception {
+        verifyGermline(".germline.vcf.gz.tbi", "reference.germline.vcf.gz.tbi", "/germline_caller/");
+    }
```

Comment:
Do we need to add a test for a file that's in a germline namespace but which doesn't have the string "germline" in its name? Does any such file exist?